tests/queries/0_stateless/02458_insert_select_progress_tcp.python

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+
+import socket
+import os
+import uuid
+import json
+
+CLICKHOUSE_HOST = os.environ.get('CLICKHOUSE_HOST', '127.0.0.1')
+CLICKHOUSE_PORT = int(os.environ.get('CLICKHOUSE_PORT_TCP', '900000'))
+CLICKHOUSE_DATABASE = os.environ.get('CLICKHOUSE_DATABASE', 'default')
+
+def writeVarUInt(x, ba):
+    for _ in range(0, 9):
+
+        byte = x & 0x7F
+        if x > 0x7F:
+            byte |= 0x80
```

Comment:
🤔 
may be add a comment why this it is done this way?
not obvious calculations...

Reply:
This was a copy-paste from `02010_lc_native.python`:

```
$ gg 'byte |= 0x80'
src/IO/VarInt.h:            byte |= 0x80;
src/IO/VarInt.h:            byte |= 0x80;
src/IO/VarInt.h:            byte |= 0x80;
tests/queries/0_stateless/02010_lc_native.python:            byte |= 0x80
```

Reply:
And neither of this code has a comment for this, so I assume that it was considered as more or less generic?

Reply:
Ok..